array-manipulation-1/src/tail.js

```diff
@@ -0,0 +1,7 @@
+/* exported tail */
+const tail = (array) => {
+  // using rest collection
+  const [x, ...result] = array;
+  console.log(x);
```

Comment:
`console.log` is only used for testing / debugging purposes, so they shouldn't be included when you submit your exercise. Only if we specifically ask for them, otherwise, they should be removed!

Reply:
![image](https://github.com/JimmyTran1206/c0523-code-solutions/assets/67088234/ff0dd369-d783-416e-9ea7-53b8b3728cdd)
thank you Shawn. I discarded the console.log() and modified the code